cli/utils/customizable-tsconfig-options.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * The allowed Typescript compiler options for consumers to override.
+ * [Typescript Compiler Options](https://www.typescriptlang.org/docs/handbook/compiler-options.html)
+ */
+const CUSTOMIZABLE_TSCONFIG_COMPILER_OPTIONS = [
+  'esModuleInterop',
+  'allowSyntheticDefaultImports',
+  // Typescript Strict Mode enables the following flags
+  'strict',
+  'noImplicitAny',
```

Comment:
This is actually [forced enabled](https://github.com/blackbaud/skyux-sdk-builder/blob/e1235701c7430a031c796fb95d4fdbf3abc2999c/cli/utils/run-build.js#L44) for SPAs already so maybe we don't allow this to be configurable?